JL.Windows/GUI/PopupWindow.xaml.cs

```diff
@@ -508,6 +518,7 @@ public void DisplayResults(bool generateAllResults)
         }
 
         PopupListView.ItemsSource = popupItemSource;
+        _ = CheckResultForDuplicates();
```

Comment:
```suggestion
       if (checkForDuplicateCards)
       {
           _ = CheckResultForDuplicates();
       }
```

Reply:
Seems VS Code isn't smart enough to know that duplicateIcons wouldn't be null... changing to this to avoid the warning

```
if (checkForDuplicateCards && duplicateIcons is not null)
{
    _ = CheckResultForDuplicates(duplicateIcons);
}
```